ts/client/src/client.ts

```diff
@@ -829,6 +830,85 @@ export class MangoClient {
     );
   }
 
+  public async emptyAndCloseMangoAccount(
+    group: Group,
+    mangoAccount: MangoAccount,
+  ): Promise<TransactionSignature> {
+    const instructions: TransactionInstruction[] = [];
+    const healthAccountsToExclude: PublicKey[] = [];
+
+    for (const serum3Account of mangoAccount.serum3Active()) {
```

Comment:
you want to cancel all serum3 orders also

Reply:
We guide user to do it by them selfs like in v3 https://pbs.twimg.com/media/FJU_ub3XoA0AEdn.png